contract/package.json

```diff
@@ -33,7 +33,7 @@
     "@endo/eslint-plugin": "^0.5.2",
     "@endo/nat": "^4.1.27",
     "@endo/promise-kit": "^0.2.56",
-    "@endo/ses-ava": "^1.4.0",
+    "@endo/ses-ava": "^1.2.4",
```

Comment:
this bumps ses-ava down/back, right? I'm interested to know why.

Reply:
1.2.4 is actually the latest version of https://www.npmjs.com/package/@endo/ses-ava